#! /bin/sh /usr/share/dpatch/dpatch-run
## 06-disable_runpath.dpatch by Sven Mueller <debian@incase.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: disable runpath

@DPATCH@
diff -urNad git~/cmulocal/cyrus.m4 git/cmulocal/cyrus.m4
--- git~/cmulocal/cyrus.m4	2010-01-16 19:21:09.000000000 -0200
+++ git/cmulocal/cyrus.m4	2010-01-16 19:25:57.999091384 -0200
@@ -3,6 +3,7 @@
 dnl $Id: cyrus.m4,v 1.5 2009/03/31 04:09:47 brong Exp $
 dnl tjs@andrew.cmu.edu 6-may-1998
 dnl
+dnl Debian fix (hmh@debian.org): We don't want runpath set in Debian
 
 dnl It would be good if ANDREW_ADD_LIBPATH could detect if something was
 dnl already there and not redundantly add it if it is.
@@ -32,14 +33,5 @@
 dnl runpath initialization
 AC_DEFUN([CMU_GUESS_RUNPATH_SWITCH], [
    # CMU GUESS RUNPATH SWITCH
-  AC_CACHE_CHECK(for runpath switch, andrew_cv_runpath_switch, [
-    # first, try -R
-    SAVE_LDFLAGS="${LDFLAGS}"
-    LDFLAGS="-R /usr/lib"
-    AC_TRY_LINK([],[],[andrew_cv_runpath_switch="-R"], [
-  	LDFLAGS="-Wl,-rpath,/usr/lib"
-    AC_TRY_LINK([],[],[andrew_cv_runpath_switch="-Wl,-rpath,"],
-    [andrew_cv_runpath_switch="none"])
-    ])
-  LDFLAGS="${SAVE_LDFLAGS}"
-  ])])
+    andrew_cv_runpath_switch="none"
+])
